packages/amplify-frontend-ios/lib/amplify-config-helper.js

```diff
@@ -0,0 +1,167 @@
+function generateConfig(context, amplifyConfig) {
```

Comment:
Can we've an ```amplify-frontend-common``` package to move these common helpers there? we're maintaining multiple copies of it. Same comments apply from above.

Reply:
Both you and Yathi brought up code re-use in  the comments of this and the other PRs. I agree with you guys in general. This can be a bigger discussion on the plugin platform refactoring on implementing common packages that will be shared by multiple plugins. Can we push this to future implementation?